MSAL/MSAL.xcodeproj/project.pbxproj

```diff
@@ -4717,7 +4719,7 @@
 			buildSettings = {
 				CODE_SIGN_IDENTITY = "-";
 				DEVELOPMENT_TEAM = "";
-				MACOSX_DEPLOYMENT_TARGET = 10.11;
+				MACOSX_DEPLOYMENT_TARGET = 10.12;
```

Comment:
can you make sure our CocoaPods file is also declaring 10.12?

Reply:
Addressed.